src/mock/class_addressGenerator.py

```diff
@@ -0,0 +1,109 @@
+"""
+A thread for creating addresses
+"""
+# import queues
+try:
+    import queue as Queue
+except ImportError:
+    import Queue
+
+import logging
+import random
+from pybitmessage import state
+import threading
+
+from pybitmessage.bmconfigparser import BMConfigParser
+
+# from network.threads import StoppableThread
+
+
+fake_addresses = {
+    'BM-2cUgQGcTLWAkC6dNsv2Bc8XB3Y1GEesVLV': {
+        'privsigningkey': '5KWXwYq1oJMzghUSJaJoWPn8VdeBbhDN8zFot1cBd6ezKKReqBd',
+        'privencryptionkey': '5JaeFJs8iPcQT3N8676r3gHKvJ5mTWXy1VLhGCEDqRs4vpvpxV8'
+    },
+    'BM-2cUd2dm8MVMokruMTcGhhteTpyRZCAMhnA': {
+        'privsigningkey': '5JnJ79nkcwjo4Aj7iG8sFMkzYoQqWfpUjTcitTuFJZ1YKHZz98J',
+        'privencryptionkey': '5JXgNzTRouFLqSRFJvuHMDHCYPBvTeMPBiHt4Jeb6smNjhUNTYq'
+    },
+    'BM-2cWyvL54WytfALrJHZqbsDHca5QkrtByAW': {
+        'privsigningkey': '5KVE4gLmcfYVicLdgyD4GmnbBTFSnY7Yj2UCuytQqgBBsfwDhpi',
+        'privencryptionkey': '5JTw48CGm5CP8fyJUJQMq8HQANQMHDHp2ETUe1dgm6EFpT1egD7'
+    },
+    'BM-2cTE65PK9Y4AQEkCZbazV86pcQACocnRXd': {
+        'privsigningkey': '5KCuyReHx9MB4m5hhEyCWcLEXqc8rxhD1T2VWk8CicPFc8B6LaZ',
+        'privencryptionkey': '5KBRpwXdX3n2tP7f583SbFgfzgs6Jemx7qfYqhdH7B1Vhe2jqY6'
+    },
+    'BM-2cX5z1EgmJ87f2oKAwXdv4VQtEVwr2V3BG': {
+        'privsigningkey': '5K5UK7qED7F1uWCVsehudQrszLyMZxFVnP6vN2VDQAjtn5qnyRK',
+        'privencryptionkey': '5J5coocoJBX6hy5DFTWKtyEgPmADpSwfQTazMpU7QPeART6oMAu'
+    }
+}
+
+
+UISignalQueue = Queue.Queue()
+addressGeneratorQueue = Queue.Queue()
+
+
+class StoppableThread(threading.Thread):
```

Comment:
Can't we import `StoppableThread` from the original location?

Reply:
we can import it but we need to deal with lots of import issues and currently python3 porting work is in progress so if I will make changes then it can cause issues so I am making StoppableThread in a separate file for a temporary basis

Reply:
I was worried about that one. How much work is it to make the original `StoppableThread` compatible with python3, can someone else help you maybe?

Reply:
We need to change the import format of the network, storage, and pyelliptic modules, and maybe there is more but when I was working on it I kept getting issues in every files one by one.